plugins/Transitions/API.php

```diff
@@ -291,7 +298,7 @@ protected function queryFollowingActions($idaction, $actionType, LogAggregator $
         $types[Action::TYPE_OUTLINK] = 'outlinks';
         $types[Action::TYPE_DOWNLOAD] = 'downloads';
 
-        $rankingQuery = new RankingQuery($limitBeforeGrouping ? $limitBeforeGrouping : $this->limitBeforeGrouping);
+        $rankingQuery = new RankingQuery($limitBeforeGrouping ?? $this->limitBeforeGrouping);
```

Comment:
That actually changes the result. The Null Coalescing Operator (`??`) is shorthand for `isset($x) ? $x : $y`.
So while before a `0` value for `$limitBeforeGrouping` was resulting in using `$this->limitBeforeGrouping` it will now result in using `0`.